pkg/operator/controller/status/controller.go

```diff
@@ -79,14 +79,14 @@ func New(mgr manager.Manager, config operatorconfig.Config) (controller.Controll
 
 // Reconcile computes the operator's current status and therefrom creates or
 // updates the ClusterOperator resource for the operator.
-func (r *reconciler) Reconcile(request reconcile.Request) (reconcile.Result, error) {
+func (r *reconciler) Reconcile(ctx context.Context, request reconcile.Request) (reconcile.Result, error) {
 	nsManifest := manifests.DNSNamespace()
 
 	co := &configv1.ClusterOperator{ObjectMeta: metav1.ObjectMeta{Name: operatorcontroller.DNSClusterOperatorName().Name}}
 	if err := r.client.Get(context.TODO(), operatorcontroller.DNSClusterOperatorName(), co); err != nil {
```

Comment:
Missed one!  We can get this in a follow-up.

Reply:
Darn!

Reply:
This didnt make FF so might as well get it now.